app/posts/claim-funding-for-mentors/2024-12-02-updating-the-files-we-send-to-providers-to-assure-claims.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Updating the files we send to providers to assure claims
+description: Details of how we iterated the comma separated value (CSV) file for providers to match their needs
+date: 2024-12-02
+tags:
+  - research
+  - accredited providers
+  - sampling
+related:
+  items:
+  - text: Round 6 accredited provider research findings
+    href: /claim-funding-for-mentors/round-6-accredited-provider-research-findings/
+---
+
+## Introduction
+
+[Round 6 accredited provider research](/claim-funding-for-mentors/round-6-accredited-provider-research-findings/) highlighted that the information provided in the first iteration of the comma separated value (CSV) file was insufficient for them to assure their claims.
+
+## CSV contents
+
+The CSV that we shared with providers contained the following information:
+
+- claim reference
+- school name
+- school unique reference number (URN)
+- claim hours
+
+The CSV included two additional columns so that the providers could assure each claim and add information into each column:
+
+- claim assured (providers to indicate yes/true or no/false)
+- claim not assured reason (providers to indicate a reason if they had marked the claim as not assured)
+
+## Research findings
+
+During round 6 we shared a CSV with the providers for them to review whether the information matched their needs so that they could accurately assure the claims.
```

Comment:
```suggestion
During round 6, we shared a CSV with the providers so that they could review whether the information matched their needs so that they could accurately assure the claims.
```